src/components/forms/form-changeprofile/index.ts

```diff
@@ -12,32 +12,25 @@ import { ImageAvatar } from "../../avatars/img-avatar/img-avatar";
 import { defaulAvatar, filePrefix } from "../../../consts";
 
 export function changeProfilePage() {
-  let inputDummy = data.input.change_profile;
+  type Inputs = {
+    name: string;
+    type: string;
```

Comment:
Тут можно ограничить вводимые значения, чтобы случайно не допустить опечатку в типе
```suggestion
    type: 'number' | 'text' | 'email' и т.д.;
```

Reply:
Сделала